pullrequest.rst

```diff
@@ -93,6 +93,89 @@ command (after any successful build of Python)::
    python.bat Tools/scripts/patchcheck.py
 
 
+Quick Guide
+'''''''''''
+
+`Clear communication`_ is key to contributing to any project, especially an
+`Open Source`_ project like Python.
+
+Here is a quick overview of you can contribute to CPython on Github:
+
+1.  `Create an Issue`_ that describes your change, if it doesn't exist and you're explicitly working on something
+
+2.  `Get started`_ and set up your system
+
+3.  Fork `CPython on Github`_ (using the Fork button in the upper-right on Github)
+
+4.  `Compile Python`_ on your system
+
+5.  `Run tests`_ after you have built Python
+
+6.  `Add an "upstream" Remote in Git`_ on your system 
+
+7.  `Create a Branch in Git`_ where you can work on changes
+
+8.  Commit changes and `resolve conflicts in Git`_
+
+9.  `Run tests`_ again
+
+10. `Push commits`_ to your Github repo
+
+11. `Create Pull Request`_ in Github to merge a branch from your Fork
+
+12. Celebrate! You just contributed to Python. :)
```

Comment:
Perhaps before celebrating the user should address the review comments and update the PR until it gets merged.